draft-ietf-quic-recovery.md

```diff
@@ -1101,22 +1101,24 @@ OnAckReceived(ack, pn_space):
 
 
 UpdateRtt(latest_rtt, ack_delay):
+  if (smoothed_rtt == 0):
+    // First RTT sample.
+    min_rtt = latest_rtt
+    smoothed_rtt = latest_rtt
+    rttvar = latest_rtt / 2
+    return
+
   // min_rtt ignores ack delay.
   min_rtt = min(min_rtt, latest_rtt)
   // Limit ack_delay by max_ack_delay
   ack_delay = min(ack_delay, max_ack_delay)
-  // Adjust for ack delay if it's plausible.
+  // Adjust for ack delay if plausible.
   adjusted_rtt = latest_rtt
-  if (latest_rtt - min_rtt > ack_delay):
+  if (latest_rtt > min_rtt + ack_delay):
     adjusted_rtt = latest_rtt - ack_delay
```

Comment:
You can simply do `latest_rtt -= ack_delay` and then use `latest_rtt` instead of `adjusted_rtt` below, since `latest_rtt` is not used anywhere anymore

Reply:
(But if you want to use the global here, maybe that adjustment to the global is wrong?)

Reply:
The prose relies on latest_rtt in descriptions.  Adjusting that value directly would confuse things.